flink-core/src/main/java/org/apache/flink/configuration/ConfigConstants.java

```diff
@@ -168,6 +168,11 @@
 	public static final String LIBRARY_CACHE_MANAGER_CLEANUP_INTERVAL = "library-cache-manager.cleanup.interval";
 
 	/**
+     * The config parameter defining the task manager resource profile.
+	 */
+	public static final String TASK_MANAGER_RESOURCE_PROFILE_KEY = "taskmanager.resourceProfile";
```

Comment:
`ConfigConstants` is annotated being `Public`. Thus, all elements which don't have an annotation, will inherit that annotation.